internal/controllers/ssp_controller.go

```diff
@@ -113,32 +122,55 @@ func (r *sspReconciler) setupController(mgr ctrl.Manager) error {
 	builder := ctrl.NewControllerManagedBy(mgr)
 	watchSspResource(builder)
 
-	r.areCrdsMissing = len(r.crdList.MissingCrds()) > 0
+	s.areCrdsMissing = len(s.crdList.MissingCrds()) > 0
```

Comment:
This could be missing CRDs of other Controllers too, right?

Reply:
Yes, thanks for noticing. I will open a separate issue to fix this. 
It does not break anything, but the resulting error message can be confusing.

Reply:
ACK